manifest.php

```diff
@@ -29,7 +29,7 @@
     'label' => 'Data Revision Control',
     'description' => 'Allows saving the intermediate state of objects and restoring them',
     'license' => 'GPL-2.0',
-    'version' => '8.6.0',
+    'version' => '8.7.0',
     'author' => 'Open Assessment Technologies SA',
     'requires' => [
         'generis' => '>=12.15.0',
```

Comment:
It is missing taoQtiItem dependency here.

- [ ] Add missing dependency

Reply:
it think the feature works because usually we have taoQtiItem inside composer , but this is correct , we should show the full list of dependencies

Reply:
Should not we use specific version? @bartlomiejmarszal  https://github.com/oat-sa/extension-tao-itemqti/commit/9cf3430b04c85eb9332874a29e9e5428a9458bf8

Reply:
@bartlomiejmarszal please use minimal needed version